### PR TITLE
Beacd-213

### PR DESCRIPTION
**Pull request by prashantchoudhari**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -508,10 +508,10 @@ usePAYEEmployerAccount.p1 = Dylech reoli CIS yn yr un cyfrif ag yr ydych yn rheo
...
 doesBusinessManagePAYE.title = A yw eich busnes yn rheoli TWE i gyflogwyr ar-lein?
 doesBusinessManagePAYE.heading = A yw eich busnes yn rheoli TWE i gyflogwyr ar-lein?
-doesBusinessManagePAYE.p1 = Bydd gennych gyfrif CThEM ar-lein fel yr un hwn, ond bydd yn defnyddio ID Defnyddiwr a chyfrinair gwahanol
-doesBusinessManagePAYE.Yes = Ydym - rydym yn rheoli TWE i gyflogwyr ar-lein
-doesBusinessManagePAYE.No = Nac ydym - nid ydym yn rheoli TWE i gyflogwyr ar-lein
-doesBusinessManagePAYE.error.required = Dewis iawn neu na
+doesBusinessManagePAYE.p1 = Os ydych, cewch gyfrif ar-lein arall gyda CThEM, fel yr un hwn, gyda Dynodydd Defnyddiwr (ID) a chyfrinair gwahanol.
+doesBusinessManagePAYE.Yes = Iawn
+doesBusinessManagePAYE.No = Na
+doesBusinessManagePAYE.error.required = Dewiswch 'Iawn' os yw'ch busnes yn rheoli TWE i gyflogwyr ar-lein
```
Comment (kurtlogan) [inline conf/messages.cy]: Replace apostrophe's

File: conf/messages.cy [hunk 2]
```diff
@@ -527,9 +527,9 @@ registerForPAYE.notnow = Nid wyf eisiau gwneud hyn ar hyn o bryd
 #######################################################
 isBusinessRegisteredForPAYE.title = A yw eich busnes wedi’i gofrestru ar gyfer TWE i gyflogwyr?
 isBusinessRegisteredForPAYE.heading = A yw eich busnes wedi’i gofrestru ar gyfer TWE i gyflogwyr?
-isBusinessRegisteredForPAYE.Yes = Ydy - mae’r busnes wedi’i gofrestru ar gyfer TWE i gyflogwyr
-isBusinessRegisteredForPAYE.No = Nac ydy - nid yw’r busnes wedi’i gofrestru ar gyfer TWE i gyflogwyr
-isBusinessRegisteredForPAYE.error.required = Dewis iawn neu na
+isBusinessRegisteredForPAYE.Yes = Iawn
+isBusinessRegisteredForPAYE.No = Na
+isBusinessRegisteredForPAYE.error.required = Dewiswch 'Iawn' os yw'ch busnes wedi'i gofrestru ar gyfer TWE i gyflogwyr
```
Comment (kurtlogan) [inline conf/messages.cy]: Replace apostrophes

File: conf/messages.cy [hunk 3]
```diff
@@ -618,29 +618,27 @@ whichVATServicesToAdd.error.required = Dewis gwasanaeth TAW
 #######################################################
 ##  UseEmployersPAYE
 #######################################################
-useEmployersPAYE.title = Defnyddio eich cyfrif TWE i gyflogwyr
-useEmployersPAYE.heading = Defnyddio eich cyfrif TWE i gyflogwyr
-useEmployersPAYE.subtext = Mae’n rhaid i chi reoli gwarantau ar sail cyflogaeth (ERS) yn yr un cyfrif ag yr ydych yn rheoli TWE i gyflogwyr.
-useEmployersPAYE.continue = Mewngofnodi i’ch cyfrif TWE i gyflogwyr
+useEmployersPAYE.title = Ychwanegu gwarantau ar sail cyflogaeth (ERS) at y cyfrif rydych yn ei ddefnyddio ar gyfer TWE i gyflogwyr
+useEmployersPAYE.heading = Ychwanegu gwarantau ar sail cyflogaeth (ERS) at y cyfrif rydych yn ei ddefnyddio ar gyfer TWE i gyflogwyr
+useEmployersPAYE.subtext = Mewngofnodi i'ch cyfrif arall ac ychwanegu ERS.
```
Comment (kurtlogan) [inline conf/messages.cy]: Replace apostrophes

File: conf/messages.cy [hunk 4]
```diff
@@ -618,29 +618,27 @@ whichVATServicesToAdd.error.required = Dewis gwasanaeth TAW
...
+useEmployersPAYE.continue = Mewngofnodi i'r cyfrif rydych yn ei ddefnyddio ar gyfer TWE i gyflogwyr
```
Comment (kurtlogan) [inline conf/messages.cy]: Replace apostrophes

File: conf/messages.cy [hunk 5]
```diff
@@ -618,29 +618,27 @@ whichVATServicesToAdd.error.required = Dewis gwasanaeth TAW
...
 useEmployersPAYE.notnow = Nid wyf eisiau gwneud hyn ar hyn o bryd
 
 #######################################################
 ##  AddEmployersPAYE
 #######################################################
-addEmployersPAYE.title = Ychwanegwch TWE i gyflogwyr yn gyntaf
-addEmployersPAYE.heading = Ychwanegwch TWE i gyflogwyr yn gyntaf
-addEmployersPAYE.subtext1 = Mae’n rhaid i chi reoli TWE i gyflogwyr ar-lein cyn y gallwch reoli gwarantau ar sail cyflogaeth (ERS).
-addEmployersPAYE.subtext2 = Ychwanegwch TWE i gyflogwyr i’r cyfrif hwn, wedyn dychwelyd ac ychwanegu gwarantau ar sail cyflogaeth (ERS) ar ôl hynny.
+addEmployersPAYE.title = Mae angen i chi reoli TWE i gyflogwyr ar-lein cyn i chi ychwanegu'r cynllun hwn
```
Comment (kurtlogan) [inline conf/messages.cy]: Replace apostrophes

File: conf/messages.cy [hunk 6]
```diff
@@ -618,29 +618,27 @@ whichVATServicesToAdd.error.required = Dewis gwasanaeth TAW
...
+addEmployersPAYE.heading = Mae angen i chi reoli TWE i gyflogwyr ar-lein cyn i chi ychwanegu'r cynllun hwn
```
Comment (kurtlogan) [inline conf/messages.cy]: Replace apostrophes

File: conf/messages.cy [hunk 7]
```diff
@@ -618,29 +618,27 @@ whichVATServicesToAdd.error.required = Dewis gwasanaeth TAW
...
+addEmployersPAYE.subtext = Pan fydd hwn gennych, dylech fewngofnodi i'ch cyfrif ac ychwanegu gwarantau ar sail cyflogaeth (ERS).
```
Comment (kurtlogan) [inline conf/messages.cy]: Replace apostrophes

File: conf/messages.cy [hunk 8]
```diff
@@ -618,29 +618,27 @@ whichVATServicesToAdd.error.required = Dewis gwasanaeth TAW
...
 #######################################################
 ##  RegisterEmployersPAYE
 #######################################################
-registerEmployersPAYE.title = Cofrestrwch ar gyfer TWE i gyflogwyr yn gyntaf
-registerEmployersPAYE.heading = Cofrestrwch ar gyfer TWE i gyflogwyr yn gyntaf
-registerEmployersPAYE.subtext1 = Mae angen i chi fod wedi cofrestru ar gyfer TWE i gyflogwyr cyn y gallwch reoli gwarantau ar sail cyflogaeth (ERS) ar-lein.
-registerEmployersPAYE.subtext2 = Cofrestrwch ar gyfer TWE i gyflogwyr, wedyn dychwelyd ac ychwanegu gwarantau ar sail cyflogaeth ar ôl hynny.
+registerEmployersPAYE.title = Cofrestru ar gyfer TWE i gyflogwyr cyn i chi ychwanegu’r gwasanaeth hwn
+registerEmployersPAYE.heading = Cofrestru ar gyfer TWE i gyflogwyr cyn i chi ychwanegu’r gwasanaeth hwn
+registerEmployersPAYE.subtext = Pan fyddwch wedi cofrestru gyda'r cyfrif hwn, dylech fewngofnodi ac ychwanegu gwarantau ar sail cyflogaeth.
```
Comment (kurtlogan) [inline conf/messages.cy]: Replace apostrophes

**Comment by prashantchoudhari** [comment]
@kurtlogan - FYI Please

**Pull request merged**